package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "@radar/lnrpc",
-  "version": "0.5.2-beta.5",
+  "version": "0.6-beta",
```

Comment:
`lnd-release-tag` had to be changed to `v0.6-beta` because that's the tag name, but the package version should still be `0.6.0-beta`. `0.6-beta` is an invalid version for an npm package.